### PR TITLE
Add DeepFlow to emerging projects

### PR DESCRIPTION
**Pull request by sharang**
Add DeepFlow to eBPF projects page. In terms of requirements for inclusion in the projects page:

- The project must be open source: ✅
  - DeepFlow is open sourced under the Apache 2.0 License
- The project must be using eBPF as its underlying core technology: ✅ 
  - **Any Stack**: With the **AutoMetrics** mechanism implemented by cBPF and **eBPF** technologies, DeepFlow can automatically collect RED (Request, Error, Delay) performance metrics of any application, down to every application call, covering all software technologie stacks from application to infrastructure.
  - **End to End**: DeepFlow innovatively implements the **AutoTracing** mechanism using **eBPF** technology. It automatically traces the **distributed** request chain of any microservice and infrastructure service in cloud-native environments.
  - eBPF code: https://github.com/deepflowys/deepflow/tree/main/agent/src/ebpf
- The project must be open to collaboration: ✅ 
  - Including external contributors, we now have a total of 23 contributors
- The project must be actively maintained. ✅
  - Last week we had 60 Pull Requests merged

### CODE REVIEW

File: src/pages/applications.js [hunk 1]
```diff
@@ -342,6 +343,17 @@ const emergingProjects = [
       {label: 'Website', url: 'https://www.loxilb.io/'},
     ],
   },
+  {
+    logoUrl: 'https://github.com/deepflowys/deepflow',
+    name: 'DeepFlow',
+    logo: deepflowLogo,
+    title: 'Highly Automated Observability Platform powered by eBPF',
+    description: `DeepFlow is a highly automated observability platform open sourced by YUNSHAN Networks Inc.. It is a high-performance observability engine that supports end-to-end distributed tracing, built for cloud-native application developers, and for any software stack. With new technologies such as eBPF, WASM and OpenTelemetry, DeepFlow innovatively implements core mechanisms such as AutoTracing, AutoMetrics, AutoTagging and SmartEncoding, helping developers to improve the automation level of code injection, reducing the maintanence complexity of the observability platform. With the programmability and open API of DeepFlow, developers can quickly integrate it into their observability stack.`,
```
Comment (xmulligan) [inline src/pages/applications.js:351]:
```suggestion
    description: `DeepFlow is a highly automated observability platform open sourced by YUNSHAN Networks Inc.. It is a high-performance observability engine that supports end-to-end distributed tracing, built for cloud native application developers and for any software stack. With new technologies such as eBPF, WASM, and OpenTelemetry, DeepFlow innovatively implements core mechanisms such as AutoTracing, AutoMetrics, AutoTagging, and SmartEncoding, helping developers to improve the automation level of code injection and reducing the maintanence complexity of the observability platform. With the programmability and open API of DeepFlow, developers can quickly integrate it into their observability stack.`,
```

**Comment by xmulligan** [comment]
Thanks for adding @sharang! Two comments:
1) Can you update the position so it is correct in the number of Github stars? It should be above L3AF
2) Can you update the description to focus on how DeepFlow leverages eBPF so it is more clear to the reader?

File: src/pages/applications.js (hunk 1 above)
Comment (borkmann) [reply src/pages/applications.js:351]: Also, please reword the beginning into:`DeepFlow is a highly automated open source observability platform. It contains a high-performance observability engine [...]`

Comment (borkmann) [reply src/pages/applications.js:351]: > With the AutoMetrics mechanism implemented by cBPF and eBPF technologies

What do you mean there by cBPF?

**Comment by sharang** [comment]
> Thanks for adding @sharang! Two comments:
> 
> 1. Can you update the position so it is correct in the number of Github stars? It should be above L3AF
> 2. Can you update the description to focus on how DeepFlow leverages eBPF so it is more clear to the reader?

thanks @xmulligan 
1. fixed
2. done

**Review by borkmann — APPROVED** [review]

File: src/pages/applications.js [hunk 2]
```diff
@@ -315,6 +316,17 @@ const emergingProjects = [
       {label: 'Website', url: 'https://merbridge.io/'},
     ],
   },
+  {
+    logoUrl: 'https://github.com/deepflowys/deepflow',
+    name: 'DeepFlow',
+    logo: deepflowLogo,
+    title: 'Highly Automated Observability Platform powered by eBPF',
+    description: `DeepFlow is a highly automated observability platform built for cloud native developers. Based on eBPF, DeepFlow innovatively implements an automated distributed tracing mechanism: AutoTracing. Microservice processes, service mesh sidecars, and network interfaces along the way are included as tracing spans, for every distributed transaction, without any code instrumentation. Besides, with the AutoMetrics mechanism implemented on cBPF and eBPF, DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`,
```
Comment (borkmann) [inline src/pages/applications.js:326]: Small nit:

`Besides, with the AutoMetrics mechanism implemented on cBPF and eBPF, DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`

Maybe change into:

`DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`

It's a bit unclear what to make out of the cBPF in the text there (I presume you mean old-style socket filters..).

Please also rebase once again w/o a merge commit. So it's all a single clean commit on top of latest master.

Comment (sharang) [reply src/pages/applications.js]: @borkmann thanks for your suggestions.

all done

**Pull request merged**